lib/groups/help.js

```diff
@@ -3,13 +3,18 @@ const chalk = require('chalk');
 const commandLineUsage = require('command-line-usage');
 
 class HelpGroup {
-    outputHelp(command = null) {
-        if (command !== null) {
-            const commands = require('../utils/commands').info;
+    outputHelp(subject = null, isCommand = true) {
+        if (subject !== null) {
+            const info = isCommand ? require('../utils/commands').info : require('../utils/core-flags').info;
+            const options = () => {
+                if (isCommand) {
+                    return info.find(cmd => cmd.name === subject || cmd.alias === subject);
+                }
+                return info.find(flg => flg.name === subject.slice(2) || flg.alias === subject.slice(1));
```

Comment:
```suggestion
                return info.find(flag => flag.name === subject.slice(2) || flag.alias === subject.slice(1));
```

Reply:
Made this change, also updated `cmd` to `command` too.